webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -151,7 +157,11 @@ const HostDetails = ({
         >
           {tabs &&
             tabs.map(tab => (
-              <Tab key={tab} eventKey={tab} title={tab}>
+              <Tab
+                key={tab}
+                eventKey={tab}
+                title={slotMetadata?.translations[tab] || tab}
```

Comment:
What about `slotMetadata` that doesn't have `translations` property?

Reply:
there is a fallback to `tab`

Reply:
It will not fall back there, it will cause an exception:
``` javascript
const slotMetadata = { bar: 'bar' };
// => undefined
slotMetadata?.translations['something']
//Uncaught TypeError: foo.translations is undefined
//    <anonymous> debugger eval code:1
slotMetadata?.translations['something'] || 'fallback'
//Uncaught TypeError: foo.translations is undefined
//    <anonymous> debugger eval code:1
```

Reply:
Thanks, this expression needs another optional chaining (`?.`) this should work:
```js
slotMetadata?.translations?.['something'] || 'fallback'
```